.github/actions/mount-s3/action.yml

```diff
@@ -9,8 +9,6 @@ runs:
         sudo apt install s3fs
         sudo mkdir /s3bucket
         sudo chown -R $(id -u):$(id -g) /s3bucket
-        echo $AWS_ACCESS_KEY_ID:$AWS_SECRET_ACCESS_KEY > ${HOME}/.passwd-s3fs
-        chmod 600 ${HOME}/.passwd-s3fs
```

Comment:
@michaljurecko  Z kódu to vypadá, že si umí načíst credentials z envů: https://github.com/s3fs-fuse/s3fs-fuse/blob/48e9e51f4fd9042b5a83d59aa00042fa8c68b043/src/s3fs_cred.cpp#L919-L921 Tak bych to nechal, abych nemusel řešit navíc ten session token pro tu assumnutou roli. Nebo s tím byl nějaký problém?

Reply:
uz si nepamatam, ci s tym bol problem, uvidime :slightly_smiling_face: